fuzz/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "scrypt-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2018"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+scrypt = { path = "../scrypt" }
+password-hash = { version = "0.6.0-rc.0", default-features = false, features = ["rand_core"], optional = true }
+hex = "0.4"
+base64 = "0.22.1"
+
+[features]
+default = ["simple", "std"]
+simple = ["password-hash"]
+std = ["password-hash/std"]
```

Comment:
I think the `features` section can be removed, i.e. it's worth to enable the `simple` feature unconditionally.

Reply:
Agreed. Thanks for your suggestion. Fixed.